src/app.js

```diff
@@ -1 +1,130 @@
+import './scss/app.scss';
 
+const input = document.getElementById('location');
+const btn = document.getElementById('submit-loc');
+const btnCelsius = document.getElementById('celsius');
+const btnFahren = document.getElementById('fahrenheit');
+const loader = document.getElementById('loader-container');
+
+const weatherIcon = document.getElementById('weather-icon');
+const infoContainer = document.getElementById('info-container');
+
+const locationInfo = document.querySelectorAll('.location > p');
+const tempInfo = document.querySelectorAll('.temp > div > p');
+const weatherInfo = document.querySelectorAll('.weather> div > p');
+const windSpeedInfo = document.querySelectorAll('.wind-speed > div > p');
+
+const errorDiv = document.createElement('div');
+
+let dataHub = null;
+
+async function fetchWeatherApi (location) {
+  errorDiv.innerHTML = '';
+  try{
+    const api = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${location}&APPID=3200d53ac65b442eb5f439f5613ee06c`);
+    const resp = await api.json();
+    console.log(resp)
+    return resp;
+  }catch (err) {
+    console.log('error: unable to fetch weather info');
+    loader.style.display = 'none';
+    errorDiv.innerHTML = `<h2>Unable to find weather info of ${input.value}</h2>`;
+    infoContainer.appendChild(errorDiv);
+  }
+}
+
+async function getWeatherInfo() {
+  const info = await fetchWeatherApi(input.value);
+  console.log(info);
+  try{
+    dataHub = getJSON(info);
+    displayWeatherInfo(dataHub);
+    loader.style.display = 'none';
+  }catch (err) {
+    loader.style.display = 'none';
+    console.log('Unable to find weather info');
+    loader.style.display = 'none';
+    errorDiv.innerHTML = `<h2>Unable to find weather info of ${input.value}</h2>`;
+    infoContainer.appendChild(errorDiv);
+  }
+  //getJSONOld(info)
```

Comment:
Please remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂